allenact_plugins/clip_plugin/objectnav_models.py

```diff
@@ -0,0 +1,163 @@
+"""Baseline models for use in the object navigation task.
+
+Object navigation is currently available as a Task in AI2-THOR and
+Facebook's Habitat.
+"""
+from typing import Tuple, Dict, Optional, cast
+
+import gym
+import torch
+import torch.nn as nn
+from gym.spaces.dict import Dict as SpaceDict
+
+from allenact.algorithms.onpolicy_sync.policy import (
+    ActorCriticModel,
+    LinearCriticHead,
+    LinearActorHead,
+    DistributionType,
+    Memory,
+    ObservationType,
+)
+from allenact.base_abstractions.distributions import CategoricalDistr
+from allenact.base_abstractions.misc import ActorCriticOutput
+from allenact.embodiedai.models.basic_models import RNNStateEncoder
+
+
+class CLIPObjectNavActorCritic(ActorCriticModel[CategoricalDistr]):
```

Comment:
Ideally this would subclass `ObjectNavActorCritic` or `VisualNavActorCritic` as these are the models used by the other objectnav baselines.

Reply:
Also maybe this should mention in the name that it's meant for zero shot?